src/main/scala/com/rawlabs/das/server/cache/iterator/ExpressionEvaluator.scala

```diff
@@ -297,9 +297,15 @@ object ExpressionEvaluator {
       // Strings
       case (StringVal(s1), StringVal(s2)) =>
         s1 < s2
-
-      // (Add date/time if needed)
-      case _ => false
```

Comment:
Same comment applies elsewhere.
Options may be cumbersome - but cleaner -, so exceptions would be easier.
That said, o1pro can write the option code if we do that route.
But this is not to be assumed in this component as it is mixing two different meanings.

Reply:
Yes it's using Exceptions now.